src/site/markdown/configuration.md

```diff
@@ -0,0 +1,49 @@
+
+# Configuration Options
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+
+
+
+| No | Key | Type | Description | Default Value | Since | Source |
+| --- | --- | --- | --- | --- | --- | --- |
+| 1. | `maven.build.timestamp.format` | `String` | Build timestamp format. |  `yyyy-MM-dd'T'HH:mm:ss'Z'`  | 3.0.0 | Model properties |
```

Comment:
I think that the desc should contain the time zone information.

Reply:
Yes, this is generated from the `defaultValue` above.